### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by rnbguy**

### CODE REVIEW

File: src/ssa/local_ssa.cpp [hunk 1]
```diff
@@ -218,8 +218,11 @@ void local_SSAt::get_globals(
       {
         // workaround for the problem that
         //  rhs() for a return value is always the "input" return value
+        #if 0 // --loc may be invalid
   const exprt &expr=is_return ?
           read_lhs(it->get_expr(), --loc) : read_rhs(it->get_expr(), loc);
+        #endif
+        const exprt &expr = read_rhs(it->get_expr(), loc);
```
Comment (peterschrammel) [inline src/ssa/local_ssa.cpp:225]: `const exprt &expr=read_rhs(it->get_expr(), loc);`

File: src/2ls/summary_checker_ai.cpp [hunk 2]
```diff
@@ -77,7 +77,9 @@ property_checkert::resultt summary_checker_ait::operator()(
     return property_checkert::UNKNOWN;
 #endif
 
-  property_checkert::resultt result=check_properties();
+  property_checkert::resultt result = options.get_bool_option("all-functions") ?
```
Comment (peterschrammel) [inline src/2ls/summary_checker_ai.cpp:80]:
```
property_checkert::resultt result=
  options.get_bool_option("all-functions")?
  check_properties():
  check_properties(goto_model.goto_functions.entry_point());
```

**Review by peterschrammel — CHANGES_REQUESTED** [review]
Start commit messages with capital letter.
Fix code formatting as suggested in my comments.

**Comment by peterschrammel** [comment]
Why did you close it? Amend your commits and git push --force

**Comment by rnbguy** [comment]
I thought it will create problem, since it will change commit ids. Anyway I created new pull request. Sorry.

**Pull request closed without merging**